### DISCUSSION TITLE
TipTap Close current block element

### DISCUSSION THREAD

**Discussion by duhaime** [ueberdosis/tiptap#2421] [General Questions]
I'm loving tiptap! I just have one small question. I'm using `editor.commands.setImage()` to insert a user-provided image. This inserts the image into the current paragraph, like so:

```
<p>
   Hi mom!
   <img src='cat.jpg'>
</p>
```

If I click to the right of the image, then start typing, the additional text is inserted into the same paragraph that contains the image:

```
<p>
   Hi mom!
   <img src='cat.jpg'>
   New text
</p>
```

Is there a way to close the current paragraph after inserting the image?

**Comment by cadars** [comment]
Did you try to change the `inline` option? https://tiptap.dev/api/nodes/image#inline
If that does not solve your problem, I’d look into the available schema options https://tiptap.dev/api/schema
